### ISSUE TITLE
name space clash

### ISSUE THREAD

**Issue by rgetz** [analogdevicesinc/ad937x-filter-wizard#2]
Summary:   RMS function in AD9371 Filter Wizard concflicts with existing function in MathWorks Signal Processing Toolbox function,​ leading to unexpected behavior

--
Problem: The 937x Filter Wizard has a function called "rms" defined here:
https://github.com/analogdevicesinc/ad937x-filter-wizard/blob/master/delsig_public/rms.m

This   shadows the rms function in Signal Processing Toolbox (introduced in R2012a).
https://www.mathworks.com/help/signal/ref/rms.html

leading to a name space clash when both are installed.

```
>> which -all rms    
C:\​ProgramData\​MATLAB\​SupportPackages\​R2018a\​3P.instrset\​analogdevices9371filterwizard.instrset\​ad937x-filter-wizard-master\​delsig_public\​rms.m      
C:\​Program Files\​MATLAB\​R2018a\​toolbox\​signal\​signal\​@tall\​rms.m
C:\​Program Files\​MATLAB\​R2018a\​toolbox\​signal\​signal\​rms.m     
```  

This can lead to unexpected behavior. For example,​ the "RWR Signal Estimation" example in Phased Array System Toolbox does not work with the Analog Devices RF Transceiver support package installed.

**Comment by tfcollins** [comment]
Only thing you can really do here is change the filename.  MATLAB doesn't really namespace things and you cannot call functions by path.

**Comment by rgetz** [comment]
yeah - change the filename to adi_rms, and then change all the calling function locations as well.
